.golangci.yml

```diff
@@ -25,7 +25,7 @@ linters:
     #- lll
     - misspell
     - nakedret
-    #- scopelint
+    - exportloopref
```

Comment:
Please keep these in alphabetical order ;-).

Reply:
Ah right, thanks, fixed in the latest!